docs/Debugging.md

```diff
@@ -0,0 +1,20 @@
+---
+id: debugging
+title: Debugging
+layout: docs
+category: Guides
+permalink: docs/debugging.html
+next: testing
+---
+
+## Debugging React Native Apps
```

Comment:
A deeper explanation of how the debugging proxying works would be awesome, but beyond my current knowledge. I've kept it simple for now. I hope that someone will add more information here eventually.